### ISSUE TITLE
Plan loading issues

### ISSUE THREAD

**Issue by HaNdTriX** [recurly/react-recurly#55]
**Describe the bug**

When using the following forked [codesandbox example](https://codesandbox.io/s/bold-tdd-idghq?file=/src/checkout-pricing.js) with my personal `RECURLY_PUBLIC_KEY` I am getting the following error:

![Screenshot 2020-04-07 at 11 53 34](https://user-images.githubusercontent.com/1265681/78655530-6e715400-78c6-11ea-9bed-8e3d9628e2a1.png)

**To Reproduce**

1. Open: https://codesandbox.io/s/bold-tdd-idghq?file=/src/checkout-pricing.js
2. Go to "Checkout Pricing"
3. Select a plan
4. Check the console

**Expected behavior**

- A. Should show the price
- B. Or at least show an error

**Your Environment**

- Which version of react-recurly are you using?  `@recurly/react-recurly@1.1.1
- Which version of react are you using? `react@16.13.0`
- Which browser (+version) are you using? `Chrome Version 80.0.3987.163 (Official Build) (64-bit)`

**Comment by dbrudner** [comment]
Great reporting, this looks like a bug. We'll look into this, thanks for posting.

**Comment by dbrudner** [comment]
@HaNdTriX  We started taking a look at this today and will begin working on a fix soon. I'll make sure to continue to give you updates as we learn more. Big thanks again for reporting and providing a reproduction.

**Comment by dbrudner** [comment]
@HaNdTriX Just to follow up, we have a fix for this issue and will be releasing it soon. Thanks for your patience!

**Comment by dbrudner** [comment]
@HaNdTriX we just released a fix for this in [v1.2.0](https://github.com/recurly/react-recurly/releases/tag/v1.2.0). It should be noted that we released a patch version immediately after that release and the latest version is v1.2.1.

Here's a working codesandbox with the issue fixed https://codesandbox.io/s/boring-napier-9gxog?file=/src/checkout-pricing.js

There's an issue on the codesandbox demo with the currency symbol not rendering correctly. I tested this locally and this is specific to codesandbox and should work fine outside of that environment.

Thanks again for reporting and sharing your issue!
